lib/ValueAnonymizers/Serialized.php

```diff
@@ -27,6 +27,15 @@ public function __construct(StringHashInterface $stringHash)
     public function anonymize(Value $value, array $row): AnonymizedValue
     {
         $serializedString = $value->getUnEscapedValue();
+
+        if (strpos($value->getRawValue(), '0') === 0) {
```

Comment:
When the serialized data is kept in a LONGBLOB column. The dump uses --hex-blob, and the Serialized anonymizer tries to unserialize a string like '0x123123ABCDEF'